public/less/_repository.less

```diff
@@ -1936,6 +1936,12 @@
             display: block;
         }
     }
+
+    .ui.fluid.action.input {
+        .ui.search.action.input {
+            flex: 1;
```

Comment:
@jaqra why no use "auto"?

Reply:
`flex: 1;` equals to:

- flex-grow: 1;
- flex-shrink: 1;
- flex-basis: 0;

`flex: auto;` equals to:

- flex-grow: 1;
- flex-shrink: 1;
- flex-basis: auto;

so in this case both are same but if you want i can change

Reply:
It is more redable i think ... thats all :)

Reply:
@jaqra Yes would be nice for other maintainers ...